tests/test_license.py

```diff
@@ -63,6 +65,20 @@ def test_simple_model_with_annual_licence(simple_linear_model):
     remaining -= remaining / (365 - 2)
     assert_allclose(lic.value(m.timestepper._next, si), remaining / (365 - 3))
 
+def test_annual_license_json(solver):
+    model = load_model("annual_license.json")
+
+    model.timestepper.start = "2001-01-01"
+    model.timestepper.end = "2001-01-31"
+    model.timestepper.delta = 5
+
+    rec = NumpyArrayNodeRecorder(model, model.nodes["supply1"])
+
+    model.run()
+
+    assert_allclose(rec.data[0], 200.0 / 365.0)
+    assert_allclose(rec.data[0], (200.0 - rec.data[0] * 5) / 360.0)
```

Comment:
These two tests are mathematically identical. Was the second one meant to test something else?
```
x = (200 - 5x) / 360
360x = 200 - 5x
x = 200 / 365
```

Reply:
The second one was supposed to test the second timestep. This is a mistake.